jdaviz/tests/test_show.py

```diff
@@ -1,17 +1,35 @@
 import sidecar
+import pytest
 
 
 def test_show_inline(specviz_helper):
-    res = specviz_helper.show_inline()
-
+    res = specviz_helper.show(loc='inline')
     assert res is None
 
 
 def test_show_sidecar(specviz_helper):
-    res = specviz_helper.show_in_sidecar()
+    res = specviz_helper.show(loc='sidecar')
     assert isinstance(res, sidecar.Sidecar)
 
 
-def test_show_new_tab(specviz_helper):
-    res = specviz_helper.show_in_new_tab()
-    assert isinstance(res, sidecar.Sidecar)
+def test_known_sidecar_anchors(specviz_helper):
+    anchors = ['split-right', 'split-left', 'split-top', 'split-bottom',
+               'tab-before', 'tab-after', 'right']
+    for anchor in anchors:
+        res = specviz_helper.show(loc=':'.join(['sidecar', anchor]))
+        assert isinstance(res, anchor)
```

Comment:
I don't think this will work... `anchor` is a string in the loop.

Reply:
Am I being dumb?
```
Python 3.10.5 (tags/v3.10.5:f377153, Jun  6 2022, 16:14:13) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> ':'.join(['sidecar', "split-right"]) 
'sidecar:split-right'
>>>
```

Reply:
What does `isinstance(res, anchor)` mean?

Reply:
Yes, I meant the `isinstance` line, the second argument to `isinstance` needs to be the type/class you're comparing it to, not a string.  Is there some way to access the anchor after the sidecar object is instantiated (something like `res.anchor == anchor`?  

As for the other line, that should work with the `.join`, but might be a little easier to read as `f"sidebar:{anchor}"`.

Reply:
Oh whoops, totally misread that lol. Thanks 😅